src/pyeconet/api.py

```diff
@@ -144,6 +144,9 @@ async def _get_equipment(self) -> None:
         for _location in _locations:
             # They spelled it wrong...
             for _equip in _location.get("equiptments"):
+                # Early exit if server returned error code
+                if "error" in _equip:
+                    continue
```

Comment:
Should we log this?

Reply:
Previously it did an error like this:
```
ERROR:pyeconet.equipment:Unknown equipment type state: None
```
I don't know that it's really an error. The `_equip` data from server is this:
```
{'error': 'Dynamic Ui Configuration for the product not found'}
```
It happens every time I run `test.py` but I'm not sure if it's specific to my account.

Reply:
Gotcha, then we should be okay. The top level debug log for the location should catch everything.

Reply:
Oops, too slow, I just pushed an error message. Want to keep it or not? The updated message prints the contents of the error from server.